packages/nextjs/app/builders/0x224aba5d489675a7bd3ce07786fada466b46fa0f/_components/SocialLinks.tsx

```diff
@@ -0,0 +1,42 @@
+"use client";
```

Comment:
We don't need this restriction since this compoent doesn't use any client-only mechanism. Please remove this line so we can improve the app's performance by pre-rendering the component during the compile process.